visualization/knit_table.R

```diff
@@ -1,8 +1,37 @@
+# # MIT License
+# 
+# Copyright (c) 2014-2020 Robrecht Cannoodt, Wouter Saelens
+# Copyright (c) ... Add scib authors here ...
```

Comment:
Leaving this as a todo note: need to add Marta Interlandi here.

Reply:
Done! I Added `Copyright (c) 2020 Marta Interlandi, Theis lab` because Theis lab is what is being used in the root of the repo.